bosh-lite/deployments/autoscaler.yml

```diff
@@ -0,0 +1,3507 @@
+api_server_properties:
```

Comment:
@boyang9527 this file should be generated using
```
./scripts/generate-bosh-lite-manifest \
	-c <path to cf-release deployment manifest> \
	-p ./example/property-overrides.yml
```
As it has consul certs coming from existing cf deployment.